demo/app-root.ts

```diff
@@ -94,56 +96,80 @@ export class AppRoot extends LitElement {
   fullscreenCheck(): void {
     if (this.showFullscreen && this.itemNav) {
       this.fullscreen = true;
-      // this.itemNav.viewportInFullscreen = true;
     }
   }
   /** End fullscreen */
 
+  toggleHeader() {
+    this.headerOn = !this.headerOn;
+  }
+
   /** Views */
-  get theaterBlock(): TemplateResult {
+  get headerExample(): TemplateResult {
     return html`
-      <ia-item-navigator
-        baseHost="https://archive.org"
-        .item=${this.itemMD}
-        .modal=${this.modalMgr}
-        .sharedObserver=${this.sharedObserver}
-        @ViewportInFullScreen=${this.toggleFS}
-        .viewportInFullscreen=${this.fullscreen}
-      ></ia-item-navigator>
+      <div slot="theater-header">
+        <div class="embed-link">
+          <img
+            src="https://archive.org/images/glogo-jw.png"
+            alt="glowing ia logo"
+          />
+          <a href="/details/goody"
```

Comment:
suggestion for readability purpose:
```
<a href="/details/goody">
The history of Little Goody Two-Shoes : otherwise called Mrs.
            Margery Two-Shoes ... [1766 edition]
</a>
```

Reply:
thanks! updated